W0219 23:42:12.448476 22854862328896 torch/distributed/run.py:779] 
W0219 23:42:12.448476 22854862328896 torch/distributed/run.py:779] *****************************************
W0219 23:42:12.448476 22854862328896 torch/distributed/run.py:779] Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
W0219 23:42:12.448476 22854862328896 torch/distributed/run.py:779] *****************************************
Init ok
Init ok
Init ok
Init ok
data module fine
data module fine
data module fine
data module fine
Traceback (most recent call last):
  File "src/training/training.py", line 156, in <module>
    model = ProteinReprModule(student_model_param=model_type_student, teacher_model_param=model_type_teacher,
  File "src/training/training.py", line 19, in __init__
    self.alphabet = self.selector.alphabet
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1729, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'ProteinReprModule' object has no attribute 'selector'
Traceback (most recent call last):
  File "src/training/training.py", line 156, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "src/training/training.py", line 156, in <module>
  File "src/training/training.py", line 156, in <module>
    model = ProteinReprModule(student_model_param=model_type_student, teacher_model_param=model_type_teacher,
  File "src/training/training.py", line 19, in __init__
    model = ProteinReprModule(student_model_param=model_type_student, teacher_model_param=model_type_teacher,
  File "src/training/training.py", line 19, in __init__
    model = ProteinReprModule(student_model_param=model_type_student, teacher_model_param=model_type_teacher,
  File "src/training/training.py", line 19, in __init__
    self.alphabet = self.selector.alphabet
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1729, in __getattr__
    self.alphabet = self.selector.alphabet
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1729, in __getattr__
    self.alphabet = self.selector.alphabet
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1729, in __getattr__
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'ProteinReprModule' object has no attribute 'selector'
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'ProteinReprModule' object has no attribute 'selector'
    raise AttributeError(f"'{type(self).__name__}' object has no attribute '{name}'")
AttributeError: 'ProteinReprModule' object has no attribute 'selector'
W0219 23:42:39.614411 22854862328896 torch/distributed/elastic/multiprocessing/api.py:858] Sending process 2903 closing signal SIGTERM
E0219 23:42:39.778582 22854862328896 torch/distributed/elastic/multiprocessing/api.py:833] failed (exitcode: 1) local_rank: 0 (pid: 2901) of binary: /home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/bin/python
Traceback (most recent call last):
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/bin/torchrun", line 33, in <module>
    sys.exit(load_entry_point('torch==2.4.1', 'console_scripts', 'torchrun')())
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 348, in wrapper
    return f(*args, **kwargs)
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/distributed/run.py", line 901, in main
    run(args)
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/distributed/run.py", line 892, in run
    elastic_launch(
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 133, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/cpebiosustain_gmail_com/miniforge3/envs/proteusAI/lib/python3.8/site-packages/torch/distributed/launcher/api.py", line 264, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
src/training/training.py FAILED
------------------------------------------------------------
Failures:
[1]:
  time      : 2025-02-19_23:42:39
  host      : hpcslurmkd-computenodeset-0.europe-west4-a.c.cpebiosustain.internal
  rank      : 1 (local_rank: 1)
  exitcode  : 1 (pid: 2902)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
[2]:
  time      : 2025-02-19_23:42:39
  host      : hpcslurmkd-computenodeset-0.europe-west4-a.c.cpebiosustain.internal
  rank      : 3 (local_rank: 3)
  exitcode  : 1 (pid: 2904)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2025-02-19_23:42:39
  host      : hpcslurmkd-computenodeset-0.europe-west4-a.c.cpebiosustain.internal
  rank      : 0 (local_rank: 0)
  exitcode  : 1 (pid: 2901)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
hpcslurmkd-computenodeset-0
/var/spool/slurmd/job00085/slurm_script: line 18: /home/cpebiosustain_gmail_com: Is a directory
